Source/DafnyCore/AST/Token.cs

```diff
@@ -74,10 +71,11 @@ public bool IsInherited(ModuleDefinition m) {
   /// </summary>
   public string TrailingTrivia { get; set; } = "";
 
-  public Token Next { get; set; } // The next token
+  public Token Next { get; set; } = NoToken; // The next token
```

Comment:
It turns out that setting `Next` and `Prev` causes some [tests](https://github.com/dafny-lang/dafny/actions/runs/13875968449/job/38828299614?pr=6148#step:11:882) to hang, so for now I'll revert their initial values to `null!`, and we can revisit it later.

Reply:
Thank you ❤️